Sync/Synchronizers/LoginsSynchronizer.swift

```diff
@@ -162,7 +162,9 @@ public class LoginsSynchronizer: IndependentRecordSynchronizer, Synchronizer {
             let lm = response.metadata.lastModifiedMilliseconds!
             log.debug("Applying incoming password records from response timestamped \(ts), last modified \(lm).")
             log.debug("Records header hint: \(response.metadata.records)")
-            return self.applyIncomingToStorage(logins, records: response.value, fetched: lm)
+            return self.applyIncomingToStorage(logins, records: response.value, fetched: lm) >>> effect {
+                NSNotificationCenter.defaultCenter().postNotificationName(NotificationDataLoginDidChange, object: nil)
```

Comment:
Don't use `NotificationDataLoginDidChange`! We listen for that to trigger a sync:

https://github.com/mozilla/firefox-ios/search?utf8=%E2%9C%93&q=onLoginDidChange